scripts/thorium_best_opds_feed/opdsfeed.json

```diff
@@ -0,0 +1,40 @@
+{
+    "NYPL instant classics": "https://instantclassics.librarysimplified.org/index.xml",
+    "NYPL open access": "http://oacontent.librarysimplified.org/",
+    "NYPL circulation": "https://circulation.librarysimplified.org/",
+    "library for all": "https://books.libraryforall.org/urms",
+    "framabookin": "http://framabookin.org/b/opds/",
+    "quebec library": "http://quebec.pretnumerique.ca/catalog/root.atom",
+    "montreal library": "http://montreal.pretnumerique.ca/catalog/root.atom",
+    "feedbooks": "https://www.feedbooks.com/catalog.atom",
+    "unglue.it": "https://unglue.it/api/opds/",
+    "web archive": "http://bookserver.archive.org/",
+    "smashwords": "http://www.smashwords.com/lexcycle/",
+    "atramenta": "http://www.atramenta.net/opds/catalog.atom",
+    "pragprog": "https://pragprog.com/magazines.opds",
+    "standard ebook": "https://standardebooks.org/opds/all",
+    "open edition": "http://opds.openedition.org",
+    "revues (open edition)": "http://bookserver.revues.org",
+    "books on board": "http://www.booksonboard.com/xml/catalog.atom",
+    "history state gov": "https://history.state.gov/api/v1/catalog",
+    "tuebl.ca": "http://tuebl.ca/catalog/",
+    "slucas.fr": "http://cops-demo.slucas.fr/feed.php",
+    "fbreader armenian": "http://armebooks.fbreader.org/books/index.xml",
+    "ebooks gratuits": "http://www.ebooksgratuits.com/opds/feed.php",
+    "many books": "http://manybooks.net/opds/",
+    "eforge": "http://eforge.eu/opds/_catalog/index.xml",
+    "ebook bike": "https://ebook.bike/catalog",
+    "wolnelektury": "http://www.wolnelektury.pl/opds",
+    "mek.oszk.hu": "http://bookserver.mek.oszk.hu",
+    "flibusta.net": "http://proxy.flibusta.net/opds",
+    "litres ru": "http://opds.litres.ru/",
+    "chitanka": "http://chitanka.info/catalog.opds",
+    "gitbook": "https://api.gitbook.com/opds/catalog.atom",
+    "lib rus ec": "http://lib.rus.ec/opds",
+    "zone 4 iphone": "http://www.zone4iphone.ru/catalog.php",
+    "flaschenpost": "https://flaschenpost.piratenpartei.de/catalog/",
+    "anarchist lib": "https://theanarchistlibrary.org/opds",
+    "aozora": "http://aozora.textlive.net/catalog.opds",
+    "prestigio plaza": "http://ebooks.prestigioplaza.com/feed",
+    "bnf.fr (gallica)": "http://gallica.bnf.fr/opds"
+}
```

Comment:
TODO @llemeurfr Curate an acceptable list of OPDS feeds (some of these are a bit, erm ... let's say "dodgy" :)

Original source:
https://github.com/readium/r2-streamer-js/blob/develop/docs/opds.md#a-selection-of-public-opds-feeds
(please contribute additions / modifications there as well, as this long list is great for testing edge-cases, including network errors, incomplete HTTP headers, invalid OPDS format, etc.)